Gems/Camera/Code/Source/EditorCameraComponent.h

```diff
@@ -53,12 +53,15 @@ namespace Camera
             AzFramework::DebugDisplayRequests& debugDisplay) override;
 
         /// EditorCameraViewRequestBus::Handler interface
```

Comment:
uber nit: Could update this to `// EditorCameraViewRequestBus overrides ...` style (now the preferred approach in the API guidelines)